front/app/components/Sharing/SharingDropdownButton/SharingDropdownContent.tsx

```diff
@@ -173,15 +190,34 @@ const SharingDropdownContent = ({
     </Email>
   );
 
+  const copylink = (
+    <CopyLink
+      setLinkCopied={setLinkCopied}
+      copyLink={url}
+      className={`sharingButton copylink`}
+    >
+      <>
+        <CopyLinkIcon ariaHidden name="link" />
```

Comment:
I think you can add an icon to the Button directly as a prop, instead of nesting it inside?